tsconfig.json

```diff
@@ -25,6 +25,7 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
     "paths": {
       "@/chainspec/*": ["./packages/chainspec/src/*"],
+      "@/scale/*": ["./packages/scale/src/*"],
```

Comment:
could this cause problems when publishing?

Reply:
no because tsup can't resolve this. it's only intended to use in tests but we'll see if that gets abused and we can remove it if so